lib/p2p/Pool.ts

```diff
@@ -114,6 +114,15 @@ class Pool extends EventEmitter {
   public broadcastOrder = (order: OutgoingOrder) => {
     const orderPacket = new OrderPacket(order);
     this.peers.forEach(peer => peer.sendPacket(orderPacket));
+
+    // TODO: send only to peers which accepts the pairId
```

Comment:
One of the first things we should address after the prealpha is adding the supported pairs to the `Hello` packet and implementing it because we have quite a bunch of `TODO` comments about that.